### DISCUSSION TITLE
AVM Terraform Pattern

### DISCUSSION THREAD

**Discussion by ChrisChapman-gh** [Azure/ipam#264] [Ideas]
Hi Folks,

I've proposed an AVM pattern module for deploying Azure IPAM.
I've got a (currently private) repo with all the code in it ready to contribute once it get approved.

https://github.com/Azure/Azure-Verified-Modules/issues/913 

At the moment it's looking for a MS employee to adopt it an act as the owner so if anyone from MS fancies picking up the mantle please let the AVM folks know.

Either way I'm working to open the repo to the public asap (hosted in company org so have to get it cleared) 

Cheers

**Comment by DCMattyG** [comment]
Hi there @ChrisChapman-gh, this is awesome news! Let me reach out internally and see what we can do.

Looking forward to seeing this as an official **Azure Verified Module** 🎉

**Reply by ChrisChapman-gh** [reply]
Thanks for the buy-in Matt, it's much appreciated.

I'm sure others will crop up but one question I did have was around the RBAC for the user assigned identity for the engine.
I see why it needs the "Managed Identity Operator" role, but I'm trying to get my head around what it needs "Contributor" for.

would you mind giving me a quick rational?

Cheers
